lib/elixir/test/elixir/kernel/errors_test.exs

```diff
@@ -660,12 +660,10 @@ defmodule Kernel.ErrorsTest do
   end
 
   test "doc attributes format" do
-    message = "expected moduledoc attribute given in the {line, doc} format, got: \"Other\""
+    message = "expected @moduledoc to be a binary, a boolean, or nil, got: 'foo'"
     assert_raise ArgumentError, message, fn ->
       defmodule DocAttributesFormat do
-        @moduledoc "ModuleTest"
-        {666, "ModuleTest"} = Module.get_attribute(__MODULE__, :moduledoc)
```

Comment:
I think we need to move this test elsewhere then (a test that get_attribute for moduledoc returns `{line, doc}`).